_maps/map_files/MiniStation/job_changes.dm

```diff
@@ -0,0 +1,150 @@
+#define JOB_MODIFICATION_MAP_NAME "MiniStation"
+
+/datum/job/assistant // Here so assistant appears on the top of the select job list.
+
+/datum/job/captain/New()
+	..()
+	total_positions = 1
+	spawn_positions = 1
+
+/datum/job/hop/New()
+	..()
+	total_positions = 1
+	spawn_positions = 1
+
+/datum/job/cargo_tech/New()
+	..()
+	total_positions = 3
+	spawn_positions = 3
+
+/datum/job/engineer/New()
+	..()
+	total_positions = 4
+	spawn_positions = 4
+
+/datum/job/doctor/New()
+	..()
+	total_positions = 4
+	spawn_positions = 4
+
+/datum/job/chemist/New()
+	..()
+	total_positions = 1
+	spawn_positions = 1
+
+
+/datum/job/scientist/New()
+	..()
+	total_positions = 4
+	spawn_positions = 4
+
+/datum/job/detective/New()
+	..()
+	total_positions = 1
+	spawn_positions = 1
+
+/datum/job/officer/New()
+	..()
+	total_positions = 4
+	spawn_positions = 4
+
+/datum/job/cyborg/New()
+	..()
+	total_positions = 1
+	spawn_positions = 1
+
+/datum/job/ai/New()
+	..()
+	total_positions = 1
+	spawn_positions = 1
+
+
+//Removed
+
+/datum/job/hydro/New()
+	..()
+	total_positions = 0
+	spawn_positions = 0
+
+/datum/job/qm/New()
+	..()
+	total_positions = 0
+	spawn_positions = 0
```

Comment:
Removed jobs should use the `MAP_REMOVE_JOB` macro. See https://github.com/tgstation/tgstation/blob/master/code/__DEFINES/maps.dm

Reply:
MAP_REMOVE_JOB macro had a couple of bugs which have been addressed. The macro itself in https://github.com/tgstation/tgstation/blob/master/code/__DEFINES/maps.dm was missing a semicolon at the end of the line (unless I am using it in an unintended way?).
Removing department heads broke late round joining and I have included a mostly fix for this, detailed in https://github.com/tgstation/tgstation/pull/48788/commits/3847d2f55a91f0a60510635db88a047a1c456ccb. Although in my opinion, having late round join list be based on a dynamic list of jobs in departments instead of static would be more future proof.